### ISSUE TITLE
Question about t2m performance metric on HumanML3D

### ISSUE THREAD

**Issue by wizardbob** [RobinWitch/SynTalker#11]
Hi, can the code support t2m performance evaluation on HumanML3D?
I notice there already has some code for t2m evaluation, but when i try to modify the test_h3d.py to support t2m evaluation, i find some files missed, e.g. dataloaders.h3d_eval_gt, corresponding to code below: 
"self.gt_data = __import__(f"dataloaders.h3d_eval_gt", fromlist=["something"]).CustomDataset(self.args, "test")"

So can you update the t2m code for me to quickly reproduce the t2m metric in the paper?

**Comment by RobinWitch** [comment]
I need to search for the t2m evaluate code and model on my previous computer and servers, because the original network code was trained on SMPLH without fingers, but what I have now is SMPLX with fingers, and the fps is also different.

I will update it within 2-3 weeks.

**Comment by SiyeolJung** [comment]
Hi, thank you for your great work. I was wondering if you have plans to release the evaluation code for the HumanML3D and BEAT-X datasets, especially for the text-to-motion (T2M) and audio-to-gesture (A2G) tasks. It would be very helpful for reproducing your results and conducting fair comparisons.
